VirtualFCS/Powertrains/ParallelHybridPowerTrain.mo

```diff
@@ -1,32 +1,33 @@
 within VirtualFCS.Powertrains;
 
 model ParallelHybridPowerTrain
-  parameter Real m_powertrain(unit = "kg") = 100 + 50;
-  parameter Real V_HV_Bus(unit = "V") = 343 "Voltage of the HV Bus";
+  import SI = Modelica.Units.SI;
+  parameter SI.Mass m_powertrain = m_FC_stack + m_bat_pack;
+  parameter SI.Voltage V_HV_Bus = 343 "Voltage of the HV Bus";
   // H2 Subsystem Paramters
-  parameter Real V_tank_H2(unit = "m3") = 0.13 "H2 tank volume";
-  parameter Real p_tank_H2(unit = "Pa") = 3500000 "H2 tank initial pressure";
+  parameter SI.Volume V_tank_H2 = 0.13 "H2 tank volume";
+  parameter SI.Pressure p_tank_H2 = 3500000 "H2 tank initial pressure";
   // Fuel Cell Stack Paramters
-  parameter Real m_FC_stack(unit = "kg") = 14.3 "FC stack mass";
-  parameter Real L_FC_stack(unit = "m") = 0.255 "FC stack length";
-  parameter Real W_FC_stack(unit = "m") = 0.760 "FC stack length";
-  parameter Real H_FC_stack(unit = "m") = 0.060 "FC stack length";
-  parameter Real vol_FC_stack(unit = "m3") = L_FC_stack * W_FC_stack * H_FC_stack "FC stack volume";
-  parameter Real V_rated_FC_stack(unit = "V") = 57.9 "FC stack maximum operating voltage";
-  parameter Real I_rated_FC_stack(unit = "A") = 300 "FC stack minimum operating voltage";
-  parameter Real i_L_FC_stack(unit = "A") = 3 * I_rated_FC_stack "FC stack maximum limiting current";
-  parameter Real I_nom_FC_stack(unit = "A") = 0.25 * I_rated_FC_stack "FC stack maximum limiting current";
+  parameter SI.Mass m_FC_stack = 14.3 "FC stack mass";
+  parameter SI.Length L_FC_stack = 0.255 "FC stack length";
+  parameter SI.Breadth W_FC_stack = 0.760 "FC stack breadth";
+  parameter SI.Height H_FC_stack = 0.060 "FC stack height";
+  parameter SI.Volume vol_FC_stack = L_FC_stack * W_FC_stack * H_FC_stack "FC stack volume";
+  parameter SI.Voltage V_rated_FC_stack = 57.9 "FC stack maximum operating voltage";
+  parameter SI.Current I_rated_FC_stack = 300 "FC stack minimum operating voltage";
+  parameter SI.Current i_L_FC_stack = 1.7 * I_rated_FC_stack "FC stack maximum limiting current";
+  parameter SI.Current I_nom_FC_stack = 0.25 * I_rated_FC_stack "FC stack maximum limiting current";
   parameter Real N_FC_stack(unit = "1") = floor(V_rated_FC_stack / 0.6433) "FC stack number of cells";
```

Comment:
Have you considered declaring a constant representing the value `0.6433` that seems to be used several places? Maybe something like `final constant SI.Voltage V_FC_something = 0.6433`, but I don't know what should be the correct name of such a constant.